cookbook/common/leaf.mk

```diff
@@ -45,14 +45,24 @@ endif
 # The Flyte project that we want to register under
 export PROJECT ?= flytesnacks
 
+PATH_TO_EXAMPLE = $(shell echo "${CURDIR}" | grep -o "cookbook.*")
```

Comment:
What is this looking for? the protobuf files?

Reply:
It will return the path after cookbook from the current dir.